# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/MediaEngineWebRTCAudio.cpp
# Commit: c54cb3c10992
# Full Hash: c54cb3c109922bc15fb6d0ca67e5f2a05980c1a5
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-07-31 15:46:43
# Regressor Bug: 1493613
# File Overlap Count: 2
# Description:
#   Bug 1493613 - Move MediaStream control from DOMMediaStream to MediaStreamTrack. r=padenot
#   
#   This is inherently large, because modifying these bits of DOMMediaStream and
#   MediaStreamTrack affects all consumers and producers of all DOMMediaStreams and
#   MediaStreamTracks.
# ==============================================================================

diff -r 0b03dd9d20ac -r c54cb3c10992 dom/media/webrtc/MediaEngineWebRTCAudio.cpp
--- a/dom/media/webrtc/MediaEngineWebRTCAudio.cpp	Wed Jul 31 09:35:06 2019 +0000
+++ b/dom/media/webrtc/MediaEngineWebRTCAudio.cpp	Wed Jul 31 07:58:17 2019 +0000
@@ -495,8 +495,7 @@
 
   AudioSegment* segment = new AudioSegment();
 
-  mStream->AddAudioTrack(mTrackID, mStream->GraphRate(), segment,
-                         SourceMediaStream::ADDTRACK_QUEUED);
+  mStream->AddAudioTrack(mTrackID, mStream->GraphRate(), segment);
 
   mInputProcessing = new AudioInputProcessing(mDeviceMaxChannelCount, mStream,
                                               mTrackID, mPrincipal);